lib/theming/src/emotion10types.ts

```diff
@@ -0,0 +1,266 @@
+import { Interpolation } from '@emotion/react';
```

Comment:
it's basically just a copy-paste (adjusted one) from `@emotion/core@10`

Reply:
I added a little not telling us to remove this when releasing 7.0